servicetalk-http-api/src/main/java/io/servicetalk/http/api/ServiceDiscoveryRetryStrategy.java

```diff
@@ -26,7 +27,13 @@
  *
  * @param <ResolvedAddress> The type of address after resolution.
  * @param <E> Type of {@link ServiceDiscovererEvent}s published from {@link ServiceDiscoverer#discover(Object)}.
+ * @see io.servicetalk.concurrent.api.RetryStrategies
+ * @see SingleAddressHttpClientBuilder#retryServiceDiscoveryErrors(BiIntFunction)
+ * @deprecated Use standard mechanisms for retrying using
+ * {@link io.servicetalk.concurrent.api.Completable#retryWhen(BiIntFunction)} on the {@link Publisher} returned from
+ * {@link ServiceDiscoverer#discover(Object)} instead.
```

Comment:
This is a misleading suggestion. Users should not retry on `ServiceDiscoverer#discover(Object)` because it will prevent client from seeing SD errors. Consider pointing to the new builder overloads that take `BiIntFunction`